src/App.vue

```diff
@@ -0,0 +1,263 @@
+<script>
+import PollOption from './PollOption.vue'
+import { getTotalVoteCount, getVoteCountsPerOption, getWinningOptions } from './computations'
+import {
+  isPollEnd,
+  isPollReset,
+  isPollResume,
+  isPollStart,
+  isPollStop,
+  isPollTitleChange,
+  isPositionChange,
+  isPrivilegedUser,
+  isValidVote
+} from './messageCheckers'
+import store from './store'
+
+import tmi from 'tmi.js'
```

Comment:
Für mehr Übersich würde ich das Wiring vll. doch in main.js lassen bzw. sogar `handleMessage` in eine extra JS-Datei (ggf. auch `messageCheckers.js`) rausziehen. Dann bliebe hier wirklich nur noch übersichtliches Rendering.